easybuild/easyconfigs/s/SCons/SCons-2.3.0-ictce-5.4.0-Python-2.7.3.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
+
+name = 'SCons'
+version = '2.3.0'
+
+homepage = 'http://www.scons.org/'
+description = "SCons is a software construction tool."
+toolchain = {'name': 'ictce', 'version': '5.4.0'} 
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://prdownloads.sourceforge.net/%(namelower)s']
+
+python = 'Python'
+pyver = '2.7.3'
```

Comment:
ok, the existing Rosetta v3.5 easyconfigs have this too, so let's leave it there